real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/ui/BuggyStatusPanelTest.java

```diff
@@ -0,0 +1,40 @@
+package com.roboclub.robobuggy.ui;
+
+/**
+ * commenting this out until gui tests work on travis
+ */
+public class BuggyStatusPanelTest {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2343306827/issues/source?bid=2938665&fileBranchId=3212596#l6)